integration-libs/cds/src/profiletag/services/profile-tag-push-events.service.ts

```diff
@@ -136,7 +136,8 @@ export class ProfileTagPushEventsService {
           return (
             previouslyEmittedCategoryPage.categoryCode ===
               currentCategoryPage.categoryCode &&
-            previousRoute.semanticRoute === currentRoute.semanticRoute
+            previousRoute.navigation.semanticRoute ===
```

Comment:
Is this actually part of this ticket? 🤔

Reply:
I was forced to it, because `semanticRoute` is now nested in `navigation` object.